src/plugins/irc-events/ctcp.js

```diff
@@ -35,6 +65,13 @@ module.exports = function (irc, network) {
 			chan = lobby;
 		}
 
+		const action = ctcpResponseActions[data.type];
+		const prventFurtherAction = action ? action(data, chan) : null;
+
+		if (prventFurtherAction) {
+			return;
+		}
+
```

Comment:
this reads strange... you have a perfectly healthy if statement, why do you use an additional ternary operator?